app/controllers/users/invitations_controller.rb

```diff
@@ -1,29 +1,44 @@
 class Users::InvitationsController < Devise::InvitationsController
-  before_action :authorise_manage_team, only: %i(create new)
```

Comment:
Where are we doing this now that its removed?

Reply:
this is part of `authenticate_inviter!`, as documented below